Add ability to mount extra volumes into nodes pods

den-is opened the pull request:
### Description
Added ability to mount arbitrary extra volumes into Chrome, Firefox and Edge nodes.

### Motivation and Context
I was testing some of my applications and in parallel decided to run Selenium Hub in my Kubernetes cluster.
Selenium test code downloads files from web-app and shared caller-script had to check files integrity.
It was easy, when I was running simple selenium containers on my localhost and mounting the required Dir into the container.
In the case of Kubernetes same problem is solved using Volumes and VolumeMounts.

My change is strictly following Kubernetes mounts and volumes definitions and accepts exactly the same format as it is described in Kubernetes docs.

### Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have read the [contributing](https://selenium.dev/documentation/en/contributing/) document.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
<!--- Provide a general summary of your changes in the Title above -->

Commit:
Added ability to mount extra volumes into nodes pods

Commit:
Added ability to mount extra volumes into nodes pods

Commit:
Added ability to mount extra volumes into nodes pods

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/SeleniumHQ/docker-selenium?pullRequest=1569) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/SeleniumHQ/docker-selenium?pullRequest=1569) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: den-is<br/>:x: selenium-ci<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/SeleniumHQ/docker-selenium?pullRequest=1569) it.</sub>

Commit:
Merge branch 'volumes' of github.com:den-is/docker-selenium into volumes

selenium-ci committed:
Update tag in docs and files [skip ci]

Updating helm chart values

Added ability to mount extra volumes into nodes pods

Commit:
update Chart.yaml

Commit:
Merge branch 'volumes' of github.com:den-is/docker-selenium into volumes

den-is commented:
hey @diemol 
I'm going to close that PR because I've messed with the commit email address and am unable to fix it.
Can you please completely remove it from the Github?

I'm going to create duplicate PR with the correct info.
Will you be able to merge it if I supply it with the correct Chart version and changelog?
I was going to wait for #1562 but looks like guy has no time and still has to re-sync his work.

lorantalas commented:
@den-is I'm awaiting review, I have time to rebase for sure, it's a matter of minutes. But I don't see the point to do it before the review.
